x/oracle/types/key_native_token.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import "strings"
+
+const (
+	// NativeTokenKeyPrefix is the prefix to retrieve all NativeToken
+	NativeTokenKeyPrefix                 = "NativeToken/"
+	NativeTokenPriceKeyPrefix            = NativeTokenKeyPrefix + "price/value/"
+	NativeTokenStakerInfoKeyPrefix       = NativeTokenKeyPrefix + "stakerInfo/value/"
+	NativeTokenOperatorInfoKeyPrefix     = NativeTokenKeyPrefix + "operatorInfo/value/"
+	NativeTokenStakerListKeyPrefix       = NativeTokenKeyPrefix + "stakerList/value/"
+	NativeTokenStakerDelegationKeyPrefix = NativeTokenKeyPrefix + "stakerDelegation/value/"
+
+// NativeTokenValidatorListKeyPrefix    = NativeTokenKeyPrefix + "validatorList/value"
+)
+
+func NativeTokenStakerDelegationKey(assetID, stakerAddr string) []byte {
+	assetID = strings.Join([]string{assetID, stakerAddr}, "/")
+	return append([]byte(NativeTokenStakerDelegationKeyPrefix), []byte(assetID)...)
+}
+
+func NativeTokenStakerListKey(assetID string) []byte {
+	return append([]byte(NativeTokenStakerListKeyPrefix), []byte(assetID)...)
+}
+
+func NativeTokenStakerKey(assetID, stakerAddr string) []byte {
+	assetID = strings.Join([]string{assetID, stakerAddr}, "/")
```

Comment:
no need check the params ?
> if assetID == "" || stakerAddr == "" {
        panic("assetID and stakerAddr must not be empty")
    }